src/store/dynamodb/mod.rs

```diff
@@ -84,7 +84,8 @@ impl StorePut for DynamoDBStore {
         self.client
             .put_item()
             .table_name(&self.table_name)
-            .set_item(Some(product.into()))
+            // TODO: Can this fail?
+            .set_item(Some(to_item(product).unwrap()))
```

Comment:
We still have a unit test for that, so we tested that `to_item` can serialize a product.

Reply:
That shouldn't fail since all the fields in a product map to valid DynamoDB attributes.